### PR TITLE
feat: use firefox private api if possible

### LINKED ISSUE #1208: [Bug] Auto paste for twitter doesnt work on Firefox

### PR DESCRIPTION
**Pull request by Jack-Works**
close #1208

### CODE REVIEW

File: src/extension/injected-script/addEventListener.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/**
```
Comment (septs) [inline src/extension/injected-script/addEventListener.ts:1]: ???

**Comment by jk234ert** [comment]
How about just auto-copy the encrypted message into the system pasteboard?

**Comment by Jack-Works** [comment]
> How about just auto-copy the encrypted message into the system pasteboard?

IIRC we're already doing this

**Comment by guanbinrui** [comment]
Merged into `release/1.13.0`

**Pull request closed without merging**